src/MDMMotionAnimator.h

```diff
@@ -167,6 +167,130 @@ NS_SWIFT_NAME(MotionAnimator)
+ sequence ends. This block has no return value and takes a single Boolean argument that indicates
+ whether or not the animations actually finished before the completion handler was called." Unlike
+ UIKit's API, if the duration of the animation is 0, this block is performed immediately. This
+ parameter may be NULL.
+ */
++ (void)animateWithDuration:(NSTimeInterval)duration
+                      delay:(NSTimeInterval)delay
+                    options:(UIViewAnimationOptions)options
+                 animations:(void (^ __nonnull)(void))animations
+                 completion:(void (^ __nullable)(BOOL finished))completion;
+
+/**
+ Similar to the UIKit method of the same name with some intentional differences in behavior.
+
+ This API does not disable user interaction during animations, unlike the default behavior of
+ UIView's similar API.
+
+ Like the UIKit API, this method performs the specified animations immediately.
+
+ @param duration From UIKit's documentation: "The total duration of the animations, measured in
+ seconds. If you specify a negative value or 0, the changes are made without animating them."
+ @param delay From UIKit's documentation: "The amount of time (measured in seconds) to wait before
+ beginning the animations. Specify a value of 0 to begin the animations immediately."
+ @param options Ignored.
+ @param dampingRatio From UIKit's documentation: "The damping ratio for the spring animation as it
+ approaches its quiescent state. To smoothly decelerate the animation without oscillation, use a
+ value of 1. Employ a damping ratio closer to zero to increase oscillation."
+ @param velocity From UIKit's documentation: "The initial spring velocity. For smooth start to the
+ animation, match this value to the view’s velocity as it was prior to attachment.
+ A value of 1 corresponds to the total animation distance traversed in one second. For example, if
+ the total animation distance is 200 points and you want the start of the animation to match a view
+ velocity of 100 pt/s, use a value of 0.5."
+ @param animations From UIKit's documentation: "A block object containing the changes to commit to
+ the views. This is where you programmatically change any animatable properties of the views in your
+ view hierarchy. This block takes no parameters and has no return value. This parameter must not be
+ NULL." Supports animating additional CALayer properties beyond what UIView's similar API supports.
+ See MDMAnimatableKeyPaths for a full list of implicilty animatable CALayer properties.
+ @param completion From UIKit's documentation: "A block object to be executed when the animation
+ sequence ends. This block has no return value and takes a single Boolean argument that indicates
+ whether or not the animations actually finished before the completion handler was called." Unlike
+ UIKit's API, if the duration of the animation is 0, this block is performed immediately. This
+ parameter may be NULL.
```

Comment:
Nit: This huge block of text is hard to parse. Maybe format to make reading the headers a little easier?

```
@param completion From UIKit's documentation: "A block object to be executed when the animation
                  sequence ends. This block has no return value and takes a single Boolean 
                  argument that indicates whether or not the animations actually finished before 
                  the completion handler was called." Unlike UIKit's API, if the duration of the 
                  animation  is 0, this block is performed immediately. This parameter may be NULL.
```

Reply:
Done.